packages/modals/src/elements/DrawerModal/DrawerModal.tsx

```diff
@@ -107,8 +111,15 @@ const DrawerModalComponent = forwardRef<HTMLDivElement, IDrawerModalProps>(
           classNames="garden-drawer-transition"
           nodeRef={transitionRef}
         >
-          <StyledBackdrop {...(getBackdropProps({ isAnimated: true, ...backdropProps }) as any)}>
-            <StyledDrawerModal {...modalProps} />
+          <StyledBackdrop
+            isAnimated
+            {...(getBackdropProps(backdropProps) as HTMLAttributes<HTMLDivElement>)}
+          >
+            <StyledDrawerModal
+              {...(getModalProps() as HTMLAttributes<HTMLDivElement>)}
+              {...props}
```

Comment:
Do we want user provided props to override what `getModalProps` outputs?

Reply:
Yes. This is how we handle other container hook usages. Consider both `Modal` and `TooltipModal` for example.